poetry.lock

```diff
@@ -1,11 +1,12 @@
-# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.1.2 and should not be changed by hand.
```

Comment:
i believe using different versions can break other devs environments, could you generate it using 1.8.3?

we should also add this to the setup guide, to specifically install poetry 1.8.3